Difi.SikkerDigitalPost.Klient/XmlValidering/Miljø.cs

```diff
@@ -37,5 +37,10 @@ internal Uri UrlWithOrganisasjonsnummer(Organisasjonsnummer databehandler, Organ
             new Uri("https://meldingsformidler.nhn.digipost.no:4444/api/"),
             new CertificateChainValidator(CertificateChainUtility.ProduksjonsSertifikater())
             );
+
+        internal static Miljø Localhost => new Miljø(
+            new Uri("http://192.168.36.1:8049"),
```

Comment:
Den er litt rar, men det er alltid på den IPen at vi får VM for VMWare på, så lar den være. Det er alltid litt grafs for å finne ut hvilke av de tre interne ip-adressene for VMen som skal brukes og hvilken port. Merk at den er `internal` så ingen eksterne vil se den.